CMakeLists.txt

```diff
@@ -10,7 +10,7 @@
 
 cmake_minimum_required(VERSION 3.8)
 
-project(serac LANGUAGES CXX)
```

Comment:
Do we know where the `find_package(HDF5)` is?  It might be preferable to add an `enable_language(C)` near where that is.

Reply:
Let's merge this and investigate afterwards so @goxberry is not held up.

Reply:
#445